### ISSUE TITLE
Difference of data key in h5 file for one vs. more frames

### ISSUE THREAD

**Issue by vhinger182** [slsdetectorgroup/slsDetectorPackage#472]
##### *Describe the bug

The key for image data in h5 files is "data" if there is only a single image and "data_f00000000..." for every other case (if more than one image is aquired and saved in the h5 file. Maybe this is a wanted distinction? In any case, it is slightly annoying when writing analysis routines because it needs to be taken into account (and it needs to be known). 

##### Distribution: 

not applicable

##### Detector type or Receiver: 

Jungfrau in particular, but is probably the same for other detectors (?)

##### Software Package Version: 

6.1.1

##### Priority:

Low

##### Expected behavior

I would expect the same key regardless of the number of frames in the file.

**Comment by thattil** [comment]
Makes sense. The files have file indices per acquisition, the datasets also needn't have this. Will get rid of it and just name the datasets "data".
